src/reducers/change-fixture.js

```diff
@@ -0,0 +1,10 @@
+var CHANGE_FIXTURE = require('../actions/change-fixture.js').type;
+
+module.exports = function(state = {}, action) {
+  switch (action.type) {
+    case CHANGE_FIXTURE:
+      return action.data || {};
```

Comment:
> The initial state should probably be `{ fixture: {} }`

While I agree with what you said, this won't work because the component gets its data 
from the top level of the store.
We need to simulate the component's original environment store, or at least the part it needs. Nesting it like that would result in a reference error.
Please correct me if I'm wrong.